packages/node_modules/@ciscospark/react-container-read-receipts/src/selectors.js

```diff
@@ -1,14 +1,36 @@
+/* eslint-disable max-params */
 import {createSelector} from 'reselect';
 
 const getActivities = (state) => state.conversation.get(`activities`);
 const getParticipants = (state) => state.conversation.get(`participants`);
+const getCurrentUser = (state) => state.user.get(`currentUser`);
```

Comment:
Yea, it triggered every time the user store updated, with this, only when current user changed.